### DISCUSSION TITLE
Import more accurate data in Photon

### DISCUSSION THREAD

**Discussion by BobLamarley** [komoot/photon#842] [Q&A]
Hi guys,
I've setted up my project with this dataset :  http://download1.graphhopper.com/public/extracts/by-country-code/fr/photon-db-fr-latest.tar.bz2
But it's not very accurate on the request 'CHU B':
For example, google is responding : 
![image](https://github.com/user-attachments/assets/1a43c631-1b9f-4d75-9152-26a93ff377f2)
And my photon :
![image](https://github.com/user-attachments/assets/f9e09e4e-5706-498a-bb0a-fdfd702a09db)
I think there is a lack of data so i've found another dataset in csv format based on OSM model with a lot more of hospital data (https://www.data.gouv.fr/en/datasets/localisation-des-hopitaux-dans-openstreetmap/#/resources), how can i import it inside my photon elasticsearch database ?

**Comment by lonvia** [comment]
Photon can only import from a Nominatim database at the moment. Adding a CSV importer is possible but so low on the TODO list that it will likely only happen as a contracted work.
